m68k-amigaos.cmake

```diff
@@ -32,21 +31,18 @@ set(M68K_FPU "soft" CACHE STRING "FPU type")
 set_property(CACHE M68K_FPU PROPERTY STRINGS ${M68K_FPU_TYPES})
 
 # CRT
-set(MCRT_TYPES "nix20" "nix13" "clib2" "ixemul" "newlib")
-set(MCRT "nix20" CACHE STRING "Target std lib")
-set_property(CACHE MCRT PROPERTY STRINGS ${MCRT_TYPES})
+set(M68K_CRT_TYPES "nix20" "nix13" "clib2" "ixemul" "newlib")
+set(M68K_CRT "nix20" CACHE STRING "Target std lib")
+set_property(CACHE M68K_CRT PROPERTY STRINGS ${M68K_CRT_TYPES})
 
 # Extra flags
 set(TOOLCHAIN_CFLAGS "${M68K_CFLAGS}" CACHE STRING "CFLAGS")
 set(TOOLCHAIN_CXXFLAGS "${M68K_CXXFLAGS}" CACHE STRING "CXXFLAGS")
 set(TOOLCHAIN_LDFLAGS "${M68K_LDFLAGS}" CACHE STRING "LDFLAGS")
 set(TOOLCHAIN_COMMON "${M68K_COMMON}" CACHE STRING "Common FLAGS")
 
-if(NOT M68K_TOOLCHAIN_PATH)
-	set(TOOLCHAIN_PATH /opt/${TOOLCHAIN_PREFIX})
-else()
-	set(TOOLCHAIN_PATH ${M68K_TOOLCHAIN_PATH})
-endif()
+set(TOOLCHAIN_PATH_DEFAULT /opt/${TOOLCHAIN_PREFIX})
+set(TOOLCHAIN_PATH ${TOOLCHAIN_PATH_DEFAULT} CACHE PATH "Path to compiler, default: ${TOOLCHAIN_PATH_DEFAULT}")
```

Comment:
I was thinking of using option() here, but that seems to only work for ON/OFF type variables: https://cmake.org/cmake/help/latest/command/option.html

Reply:
Otherwise, this seems good to me. :)